packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -560,8 +560,13 @@ class CustomMultiChildLayout extends MultiChildRenderObjectWidget {
 
 /// A box with a specified size.
 ///
-/// Forces its child to have a specific width and/or height and sizes itself to
-/// match the size of its child.
+/// If given a child, this widget forces its child to have a specific width
+/// and/or height (assuming values are permitted by this widget's parent). If
+/// either the width or height is null, this widget will size itself to match
+/// the child's size in that dimension.
+///
+/// If not given a child, this widget will size itself to the given width and
+/// height, using zero for either dimension that is null.
```

Comment:
"using zero for either dimension that is null" is grammatically dubious.

Maybe "treating nulls as zero" or something?